scripts/update_teams.py

```diff
@@ -87,18 +94,15 @@ def create_team(org, name, description, repos):
     all_members.update(contributors)
 
     # Get the github repo for this feedstock.
-    repo = conda_forge.get_repo('{}-feedstock'.format(package_name))
 
     # If the team already exists, get hold of it.
     team = teams.get(package_name)
     if not team:
+        repo = conda_forge.get_repo('{}-feedstock'.format(package_name))
         team = create_team(conda_forge, package_name.lower(),
                            'The {} {} contributors!'.format(choice(superlative), package_name),
                            [repo])
         teams[package_name] = team
```

Comment:
TBH I don't think we need team creation here at all. This is a holdover from before we moved this logic to staged-recipes.

Any feedstock that we encounter should either have a team or staged-recipes failed in conversion. If the latter, we should just skip over this feedstock/team and print a message to the log.